ion/importlocation.go

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package ion
+
+// TODO comment
+type ImportLocation struct {
+	importName *string
```

Comment:
As with my feedback on #93, pointers to strings feel weird and unidiomatic in go. Most notably, it means you can't call NewImportLocation as a one-liner with an inline name, you've got to allocate a new string object on the heap. Would it be possible to have this just be a normal `string`, or will importName ever be meaningfully nil?

You also probably want to either add accessor methods for importName and sid, or (my vote) just export them and get rid of `NewImportLocation`.

Reply:
> As with my feedback on #93, pointers to strings feel weird and unidiomatic in go. Most notably, it means you can't call NewImportLocation as a one-liner with an inline name, you've got to allocate a new string object on the heap. Would it be possible to have this just be a normal string, or will importName ever be meaningfully nil?

When SymbolToken is created with default ImportLocation, it will look like: 
```
ImportLocation {
  ImportName: nil
  SID: 0
}
```

This is used to represent an undefined ImportLocation. 



> You also probably want to either add accessor methods for importName and sid, or (my vote) just export them and get rid of NewImportLocation.

Removed `NewImportLocation` and exported the struct attributes

Reply:
Thinking about this more, we can set SymbolToken ImportLocation to be a pointer and the default ImportLocation will be nil. Setting ImportLocation to a normal string